TRANSLATING.md

```diff
@@ -17,9 +17,10 @@ If you have an open issue for a language request, **add the issue link to the PR
    
 3. When you conribute to a Qiskit open source project on GitHub with a new pull request, a bot will evaluate whether you have signed the [Qiskit Contribution License Agreement (CLA)](https://qiskit.org/license/qiskit-cla.pdf). If it is your first time contributing to Qiskit, you will be prompted to sign the CLA in your PR. Please make sure to sign the CLA by accepting the agreement.<br/>
 4. Join [GitLocalize](https://gitlocalize.com) using the same GitHub ID you used to open your PR above.  
-5. Once your PR has been accepted and merged, you will receive a notification from GitLocalize with a title: `[GitLocalize] You have been assigned a new role.`<br/>
-6. Join project [Qiskit/platypus] and then visit the language you signed up to contribute to.<br/>
-7. For questions about the project, to connect with other translators, or to receive updates regarding the project, please join #qiskit-localization channel in the [Qiskit workspace](http://ibm.co/joinqiskitslack). <br/>
+5. Leave a comment in your PR stating that you successfully joined GitLocalized.
```

Comment:
@veenaiyuri - is it `GitLocalize`, or `GitLocalized`? (same thing?)

thanks!

Reply:
@techtolentino omg, you are right. It is `GitLocalize`. Thanks for catching this!